pkg/commands/artifact/run.go

```diff
@@ -579,7 +579,7 @@ func initScannerConfig(opts flag.Options, cacheClient cache.Cache) (ScannerConfi
 			Trace:                   opts.Trace,
 			Namespaces:              append(opts.PolicyNamespaces, defaultPolicyNamespaces...),
 			PolicyPaths:             append(opts.PolicyPaths, downloadedPolicyPaths...),
-			DataPaths:               opts.DataPaths,
+			DataPaths:               append(opts.DataPaths, downloadedPolicyPaths...),
```

Comment:
Why do we want to pass policy paths as data paths? The policies are loaded as data by mistake, no? I may be missing something.

Reply:
it was a typo, updated here https://github.com/aquasecurity/trivy/pull/4745/commits/18b29d5d6e399de2a46fd00ebbb61a34fd01e32d